src/components/form-elements/Input.jsx

```diff
@@ -154,13 +162,27 @@ const Input = (props: InputPropsType) => {
   );
 
   return (
-    <input
-      {...additionalProps}
-      type={type}
-      ref={setInputRef}
-      className={inputClass}
-      value={value}
-    />
+    <div>
```

Comment:
@bartoszlorek wdyt about wrapping it with `<div />` here?
maybe we should wrap it only when it has an errror message?
or it's better to always wrap it.

Reply:
`input` gets `component()` mixins and it's a `display: inline-block` so we want to keep it, not rewrite to just a `block`. I think it can be just a react `fragment` and let others decide about structure in implementation.

Reply:
agree, can we stick to `</>` @anna-wro ?

Reply:
hmmm, on the other hand `message` will be always displayed as a `block`. My only concern is when div will be full width and input inside not.

Reply:
we don't want such case 😢 
<img width="649" alt="Screen Shot 2020-03-27 at 09 31 50" src="https://user-images.githubusercontent.com/13873576/77737220-d1810200-700d-11ea-82a5-44aa9ec62dbf.png">
<img width="499" alt="Screen Shot 2020-03-27 at 09 30 17" src="https://user-images.githubusercontent.com/13873576/77737164-b615f700-700d-11ea-94aa-44980545ce2e.png">
<img width="487" alt="Screen Shot 2020-03-27 at 09 30 24" src="https://user-images.githubusercontent.com/13873576/77737165-b7dfba80-700d-11ea-9e79-0b345a51cd3f.png">

Reply:
@anna-wro 
after discussion with @bartoszlorek we think we could stick to `<div />`as a wrapper.
but not the break anything from layout perspective we need to:
1. add there a className, i.e. `sg-input__wrapper` and styles:
```
.sg-input__wrapper {
  display: inline-block;
  $--full-width {
    width: 100%;
  }
}
```
so fullWith prop would be used on both wrapper and input
2. let's wrap it only when there is an errorMessage, when there is no, let's return just an `<input />`.

@anna-wro wdyt?

Reply:
ok, I'll change it

Reply:
do we want the same with textarea?

Reply:
@bartoszlorek @matzimowski @ElaKrk Please check one more time :)